secretmanager/snippets/snippets_test.py

```diff
@@ -48,22 +49,24 @@
 
 
 @pytest.fixture()
-def client():
+def client() -> secretmanager.SecretManagerServiceClient:
     return secretmanager.SecretManagerServiceClient()
 
 
 @pytest.fixture()
-def project_id():
+def project_id() -> str:
     return os.environ["GOOGLE_CLOUD_PROJECT"]
 
 
 @pytest.fixture()
-def iam_user():
+def iam_user() -> str:
     return "serviceAccount:" + os.environ["GCLOUD_SECRETS_SERVICE_ACCOUNT"]
 
 
 @pytest.fixture()
-def secret_id(client, project_id):
+def secret_id(
+    client: secretmanager.SecretManagerServiceClient, project_id: str
+) -> Generator[str, None, None]:
```

Comment:
Is Generator right here? It looks like it's yielding a str to me. What am I missing?

Reply:
I originally tried `-> str`, but since this method doesn't return anything, that doesn't make sense. 

mypy says `The return type of a generator function should be "Generator" or one of its supertypes`

Adding just `Generator[str]` causes `"Generator" expects 3 type arguments, but 1 given`. Typing documentation for [Generator](https://docs.python.org/3/library/typing.html#typing.Generator) says you can use a triple for `[yield type, send type, return type]`, which is what I used. But that feels overly verbose

Reading further, I could just use `Iterator[str]` which looks cleaner, and mypy is happy.